CONTRIBUTING.md

```diff
@@ -138,8 +139,8 @@ New features:
 ### Devops
 New features:
 
-- If there are any new configuration properties, do the follow the same
-  structure as existing configuration style?
+- Does the configuration style follow the config guidelines? That is compile
+  (Maven) config goes in `pom.xml`, runtime (Spring) goes in `portal.properties`.
```

Comment:
The lack of clarity on that last point has already led to at least one broken view in the past, see pull request #1389. A newly introduced setting had a suggested value in `portal.properties.EXAMPLE` but no default in `GlobalProperties.java`.